### DISCUSSION TITLE
Multiple GPU training

### DISCUSSION THREAD

**Discussion by pwylcrow** [SciSharp/TensorFlow.NET#1046] [Q&A] [answered]
I'm using the Keras bindings for model training.  Is there a clear example on how to split training between two gpus?

**Comment by SanftMonster** [comment]
I'm sorry that tf.net hasn't supported distributed training now. It's on our schedule but need much time since it's a big feature. Do you want to use distributed training because of large model size?

**Reply by pwylcrow** [reply]
That's correct, I have two 4090's and I need to split training between them.  I'm a C# developer by trade and would prefer not to convert all of my code to python :(.

**Reply by SanftMonster** [reply, marked as answer]
We'll implement it in the future but now I can only give a timeline of our developing since there's a short of hands of tf.net. (ノ=Д=)ノ┻━┻

We're now on v0.100.4 and we'll publish v0.100.5 as the last version of v0.x in a week.

No later than july we'll publish v1.0.0, which provides better binding of keras and some tools such as tf hub, tf text and opencv adapter.

Then we'll maintain version v1.0.x to fix BUGs and add docs and examples. After that we'll start the next big feature development. I think it will be after August.

Thank you for your supporting for tf.net. We hope we could make it the best deep learning framework of C#. Also, welcome to join us to develop it together!

**Reply by pwylcrow** [reply]
Sure I would love to help you folks
